ApptentiveFeedback/ApptentiveFeedback/Additions/ApptentiveConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Android.Icu.Util;
+using ApptentiveKit.Android;
+using Java.Util.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using ApptentiveCore;
+using Apptentive.Com.Android.Util;
+
+namespace ApptentiveKit
+{
+    [Serializable]
+    public class ApptentiveConfiguration
+    {
+        public LogLevel LogLevel { get; set; } = LogLevel.Info;
+        public string ApptentiveKey { get; set; }
+        public string ApptentiveSignature { get; set; }
+        public bool ShouldInheritAppTheme { get; set; } = true;
+        public bool ShouldSanitizeLogMessages { get; set; } = true;  
+        public string CustomAppStoreURL { get; set; } = null;
+        private string DistributionName { get; } = "Xamarin";
+        private string DistributionVersion { get; } = "";
```

Comment:
Should this be set here or will it be set somewhere else?

Reply:
I made it private in the metadata so its only internal to the binding and not accessible to the app. Then before register is called internally i set the distribution name/version.